docs/projects/ansible/collection/centurion/playbooks/teams.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Teams
+description: No Fuss Computings Companion Ansible Collection Centurion teams Playbook.
```

Comment:
the what now??

Reply:
Still need to adjust this

Reply:
adjusted in commit [cec6cd9](https://github.com/nofusscomputing/ansible_collection_centurion/pull/17/commits/cec6cd91a382d424c008dd89b841499e6299691a)